test/e2e/case20_informonly_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	"context"
+	"errors"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"open-cluster-management.io/config-policy-controller/test/utils"
+)
+
+var _ = Describe("Test 'InformOnly' ConfigurationPolicies", Ordered, func() {
+	const (
+		case20PolicyName    string = "case20-policy-informonly"
+		case20PolicyYaml    string = "../resources/case20_policy_informonly/case20-parent-policy.yaml"
+		case20ConfigMapName string = "test-config"
+		case20ConfigMapYaml string = "../resources/case20_policy_informonly/case20-configmap.yaml"
+		case20ConfigPlcName string = "create-configmap"
+	)
+
+	BeforeEach(func() {
+		hubApplyPolicy(case20PolicyName, case20PolicyYaml)
+	})
+
+	AfterEach(func() {
+		By("Deleting a policy on the hub in ns:" + clusterNamespaceOnHub)
+		_, err := kubectlHub("delete", "-f", case20PolicyYaml, "-n", clusterNamespaceOnHub)
+		var e *exec.ExitError
+		if !errors.As(err, &e) {
+			Expect(err).ShouldNot(HaveOccurred())
+		}
+		opt := metav1.ListOptions{}
+		utils.ListWithTimeout(clientManagedDynamic, gvrPolicy, opt, 0, true, defaultTimeoutSeconds)
+	})
+
+	AfterAll(func() {
+		_, err := kubectlManaged("delete", "configmap", case20ConfigMapName, "-n",
+			clusterNamespace, "--ignore-not-found")
+		var e *exec.ExitError
+		if !errors.As(err, &e) {
+			Expect(err).ShouldNot(HaveOccurred())
+		}
+	})
+
+	Describe("Override remediationAction in spec", func() {
+		Context("When the CR does not exist", func() {
+			It("Should be noncompliant", func() {
+				Eventually(func() interface{} {
+					plc := utils.GetWithTimeout(clientManagedDynamic, gvrConfigurationPolicy,
+						case20ConfigPlcName, clusterNamespace, true, defaultTimeoutSeconds)
+
+					return utils.GetComplianceState(plc)
+				}, defaultTimeoutSeconds, 1).Should(Equal("NonCompliant"))
```

Comment:
This test needs a running configuration-policy-controller to set the compliance state on that ConfigurationPolicy. In these tests, only the framework-addon is running, so that won't work. I think the test can just be removed - we don't need to worry about the compliance in these tests, only the `remediationAction`.